### PR TITLE
New package: treesheets

### PR DESCRIPTION
**Pull request by Piraty**
The website states it required wxWidgets-3.1, but it runs fine with our wxWidgets-3.0 so far.
This software uses an unconventional, self-containing file hierarchy, hence the excessive `do_install()` and the custom `do_build()`.

The template is based on @masato9000's initial contribution to the forum (https://forum.voidlinux.eu/t/cool-apps-for-new-packagers/1642/6).
He is ok with my PR.

About versioned releases of treesheets: https://github.com/aardappel/treesheets/issues/83
(author is not in favour right now)

### CODE REVIEW

**Comment by Vaelatern** [comment]
Do you want to maintain it? Go ahead and ask to be allowed to push arbitrary version tags.

File: srcpkgs/treesheets/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+short_desc="Free Form Data Organizer"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="ZLIB"
+homepage="http://strlen.com/treesheets/"
+distfiles="https://github.com/aardappel/treesheets/archive/${_githash}.tar.gz"
+
+#build_wrksrc="src"
+
+do_build() {
+	cd $wrksrc/src
```
Comment (Vaelatern) [inline srcpkgs/treesheets/template]: Why not `build_wrksrc`?

File: srcpkgs/treesheets/template [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'treesheets'
+pkgname=treesheets
+version=20170731
```
Comment (Vaelatern) [inline srcpkgs/treesheets/template]: I tagged a release. Please use it.

File: srcpkgs/treesheets/template [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	vmkdir usr/lib/${pkgname} 755
+	vcopy ${wrksrc}/TS/docs usr/lib/${pkgname}
+	vinstall ${wrksrc}/TS/readme.html 644 usr/lib/${pkgname}
+	vcopy ${wrksrc}/TS/examples usr/lib/${pkgname}
+	vcopy ${wrksrc}/TS/images usr/lib/${pkgname}
+	vinstall ${wrksrc}/TS/treesheets 755 usr/lib/${pkgname}
+	vinstall ${wrksrc}/TS/treesheets.desktop 644 usr/share/applications
+	vinstall ${wrksrc}/TS/images/icon32.png 644 usr/share/pixmaps treesheets.png
+	vinstall ${wrksrc}/TS/images/icon16.png 644 usr/share/pixmaps treesheets-16x16.png
+	#vinstall ${wrksrc}/TS/images/treesheets.svg 644 usr/share/pixmaps treesheets.svg
```
Comment (Vaelatern) [inline srcpkgs/treesheets/template]: commented out?

**Comment by Piraty** [comment]
thanks @Vaelatern, i applied your suggestions.
`do_build()` is still necessary, the build fails without the re-definition.

**Comment by the-maldridge** [comment]
This sounds fishy as something that doesn't build without dropping all of Void's flags.  I'm somewhat suspicious of that makefile.  Can you try the build but with make_use_env=1 set (comment out your do_build() after doing that).

**Comment by Piraty** [comment]
Nice catch, thanks @the-maldridge . this obviously did the trick. PR is updated.

**Pull request closed without merging**